src/error.rs

```diff
@@ -30,6 +30,15 @@ pub enum Error {
     #[error("Nix error: {0}")]
     Nix(#[from] nix::Error),
 
+    #[error("ttrpc err: local stream closed")]
+    LocalClosed,
+
+    #[error("ttrpc err: remote stream closed")]
+    RemoteClosed,
+
+    #[error("eof")]
+    EOF,
```

Comment:
`Eof` is better, EOF will get following warning below the rust version 1.54
```
^^^ help: consider making the acronym lowercase, except the initial letter (notice the capitalization): `Eof`
```

Reply:
fixed